analytic_diffuse/__init__.py

```diff
@@ -19,7 +19,7 @@ def get_model(name):
         Callable function.
     """
     if name not in available_models:
-        raise ValueError("Model {} is not available".format(name))
+        raise ValueError(f"Model {name} is not available")
```

Comment:
Wow! I've never heard of this formatting before. Neat!

Reply:
Yup, it's the new best way to do string formatting since py3.6+.